source/titfront/src/components/Menu.tsx

```
@@ -0,0 +1,191 @@
+  icon: ReactElement<IconBaseProps>;
+  callback: () => void;
+}
+
+export interface MenuProviderProps {
+  actions: Action[];
+}
+
+const MenuContext = createContext<MenuProviderProps | null>(null);
+
+export function useMenu() {
+  return useContext(MenuContext)!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)